docs/en/ingest-management/elastic-agent/otel-agent.asciidoc

```diff
@@ -0,0 +1,145 @@
+
+When you run {agent} in `otel` mode it supports the standard OTel Collector configuration format that defines a set of receivers, processors, exporters, and connectors. Logs, metrics, and traces can be ingested using OpenTelemetry data formats.
+
+The following steps will help you to get set up quickly using {agent} as an OTel Collector. Note that all steps are run with `sudo` as it's generally recommended to run {agent} with administrative privileges.
+
+[discrete]
+== Run {agent} in OTel mode
+
+For simplicity, these steps demonstrate OTel mode using standalone {agent}. The install steps for {fleet}-managed {agent} are very similar. Refer to the <<elastic-agent-installation,Install {agents}>> section for the full set of requirements and steps for any install scenario.
+
+. On your host, download and extract the {agent} installation package. 
++
+--
+// tag::install-elastic-agent[]
+
+include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/download-widget.asciidoc[]
+
+// end::install-elastic-agent[]
+--
++
+The commands shown are for AMD platforms, but ARM packages are also available.
+Refer to the {agent} https://www.elastic.co/downloads/elastic-agent[downloads page]
+for the full list of available packages.
+
+. Change into the agent package directory and run {agent} with the `otel` subcommand and the `-h` option to view the command syntax and available parameters:
++
+[source,shell]
+--
+cd <agent-package-name>
+sudo ./elastic-agent otel -h
+--
++
+TIP: You can also refer to the <<elastic-agent-otel-command,{agent} OTel>> command documentation for the full command syntax.
+
+. View or update the OTel configuration file.
++
+A basic OTel configuration file named `otel.yml` is included in the {agent} package. You can view or update it using any text editor.
++
+[source,yaml]
+--
+receivers:
+  filelog:
+    include: [ /var/log/system.log ]
+    start_at: beginning
+
+processors:
+  resource:
+    attributes:
+    - key: service.name
+      action: insert
+      value: elastic-otel-test
+
+exporters:
+  debug:
+    verbosity: detailed
+    sampling_initial: 10000
+    sampling_thereafter: 10000
+
+service:
+  pipelines:
+    logs:
+      receivers: [filelog]
+      processors: [resource]
+      exporters:
+        - debug%
+--
++
+By default, the default OTel configuration sets {agent} to:
+
+* Read the contents of `/var/log/system.log`.
+* Run a processor to insert `elastic-otel-test` into the output.
+* Send the data with a `detailed` level of verbosity.
++
+Refer to <<otel-agent-configuration>> to learn more about setting up this file.
+
+. [Optional] Override a setting in the OTel configuration
++
+You can use the `set` flag to manually override any setting defined in the OTel configuration file. For example, run the following command to change the default verbosity setting from `detailed` to `normal`.
++
+[source,shell]
+----
+sudo ./elastic-agent otel --config otel.yml --set "exporters::debug::verbosity=normal"
```

Comment:
For 8.15 we will be including a script that allows you to run `./otelcol` instead of `./elastic-agent otel` on at least macos and windows

Reply:
Nice! I've added:

![Screenshot 2024-06-18 at 1 32 27 PM](https://github.com/elastic/ingest-docs/assets/41695641/c60d97d5-10f9-445c-93d8-c7f54a620f07)